auto_backup/models/db_backup.py

```diff
@@ -83,6 +83,13 @@ class DbBackup(models.Model):
              "read permissions for that file.",
     )
 
+    backup_format = fields.Selection(
+        [("dump.zip", "zip (includes filestore)"), ("dump", "pg_dump custom format (without filestore)")],
```

Comment:
Shouldn't be "`zip`" here? I'm wondering how this functionality could work fine with "`dump.zip`" passed to `dump_db()` :thinking:

Reply:
True!

Reply:
@astirpe fixed!